lib/ansible/modules/system/iptables.py

```diff
@@ -641,6 +654,7 @@ def main():
             chain=dict(type='str'),
             rule_num=dict(type='str'),
             protocol=dict(type='str'),
+            wait=dict(type='str'),
```

Comment:
@LinusU The type above is just a doc for this line. When I try using `int` here I get an error.

```
TASK [Remove a rule] *****************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: 'int' object has no attribute '__getitem__'
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/home/sbarre/.ansible/tmp/ansible-tmp-1563515535.33-152683851725308/AnsiballZ_iptables.py\", line 125, in <module>\n    _ansiballz_main()\n  File \"/home/sbarre/.ansible/tmp/ansible-tmp-1563515535.33-152683851725308/AnsiballZ_iptables.py\", line 117, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/sbarre/.ansible/tmp/ansible-tmp-1563515535.33-152683851725308/AnsiballZ_iptables.py\", line 54, in invoke_module\n    imp.load_module('__main__', mod, module, MOD_DESC)\n  File \"/tmp/ansible_iptables_payload_SrSlU0/__main__.py\", line 776, in <module>\n  File \"/tmp/ansible_iptables_payload_SrSlU0/__main__.py\", line 717, in main\n  File \"/tmp/ansible_iptables_payload_SrSlU0/__main__.py\", line 523, in construct_rule\n  File \"/tmp/ansible_iptables_payload_SrSlU0/__main__.py\", line 487, in append_param\nTypeError: 'int' object has no attribute '__getitem__'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

I also notice the port number fields of this module are all `str` as well.

Reply:
Figured out the error and fixed it!